spec/factories/in_person_enrollments.rb

```diff
@@ -17,7 +17,10 @@
       status { :pending }
       status_updated_at { Time.zone.now }
       profile do
-        association(:profile, :in_person_verification_pending, user: user)
+        association(
+          :profile, :in_person_verification_pending, user: user,
+                                                     in_person_enrollment: instance
```

Comment:
personal code style, I use an "all same or all different" rule for params with their own lines, I think it would make this easier to follow

```suggestion
          :profile,
          :in_person_verification_pending,
          user: user,
          in_person_enrollment: instance,
```

Reply:
Agreed, I committed the suggestion.